wg-api-expression/2020-annual-report.md

```diff
@@ -0,0 +1,38 @@
+# 2020 - WG API Expression - Community Meeting Annual Reports
+
+**What was the initial mission of the group and if it's changed, how?**
+
+Should still be the same as outlined in our [mission statement](https://github.com/kubernetes/community/blob/master/wg-api-expression/README.md), but with a hard focus on Server Side Apply.
+
+**What’s the current roadmap until completion?**
+
+- Server Side Apply (GA targeted for v1.21).
+
+    Currently the WorkGroup is mostly focused on delivering Server Side Apply.
+
+- Our Mission Statement contains various other topics we would like to work on, but finishing and delivering
+a working version of SSA is our primary goal right now.
+
+**Have you produced any artifacts, reports, white papers to date?**
+
+- SSA went to Beta 2 with K8s v1.18
+    
+    This enabled the tracking of managedFields by default on every new resource.
+[There was a Blog Post about it](https://kubernetes.io/blog/2020/04/01/kubernetes-1.18-feature-server-side-apply-beta-2/).
+
+**Is the group active? healthy? contributors from multiple companies and/or end user companies?**
+
+- The group meets actively every two weeks with communication on GitHub and Slack in between.
+- Contributors are from various companies, including Google, IBM, Red Hat and end user companies.
+
+**Is everything in your readme accurate? posting meetings on youtube?**
+
+- The readme should be accurate.
+- We’re uploading videos to youtube.
+
+**Do you have regular check-ins with your sponsoring SIGs?**
+
+I wouldn’t say regular check-ins, but people from those SIGs are present at most meetings.
+If we should have a more formal check-in, we can figure something out.
```

Comment:
With SSA in mind, we've tried to check-in a few times since the beginning of the release, there's no format guidelines/timeline though, I agree.